git_autoshare/clone.py

```diff
@@ -8,31 +8,36 @@
 import subprocess
 import sys
 
-from .core import git_bin, prefetch_one, shared_urls
+from .core import git_bin, prefetch_one, _repo_cached
 
 
 def main():
     cmd = [git_bin(), 'clone'] + sys.argv[1:]
-    skip = \
-        '--reference' in cmd or \
-        '--reference-if-able' in cmd or \
-        '-s' in cmd or \
-        '--share' in cmd
+    skip = any(
+        [c in cmd for c in [
```

Comment:
you don't need the list comprehension as `any` can take a generator
```suggestion
        c in cmd for c in [
```